{
  "uid" : "6b6710fd0f020703",
  "name" : "test_login",
  "fullName" : "tests.test_login#test_login",
  "historyId" : "144503cdd62308bf9f77a1ff306826fa",
  "time" : {
    "start" : 1627369152663,
    "stop" : 1627369153140,
    "duration" : 477
  },
  "status" : "broken",
  "statusMessage" : "FileNotFoundError: [Errno 2] No such file or directory: 'login.csv'",
  "statusTrace" : "def test_login():\n        try:\n            driver.get('http://localhost:1667/#/login')\n    \n            def find_and_clear(element_xpath):\n                element = driver.find_element_by_xpath(element_xpath)\n                element.clear()\n                return element\n    \n            sign_in_button = driver.find_element_by_xpath('//*[@id=\"app\"]/div/div/div/div/form/button')\n    \n            # data entry:\n>           with open('login.csv', 'r') as login_data:\nE           FileNotFoundError: [Errno 2] No such file or directory: 'login.csv'\n\ntests/test_login.py:33: FileNotFoundError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "tests"
  }, {
    "name" : "suite",
    "value" : "test_login"
  }, {
    "name" : "host",
    "value" : "fv-az74-91"
  }, {
    "name" : "thread",
    "value" : "3554-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.test_login"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Test defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "6b6710fd0f020703.json",
  "parameterValues" : [ ]
}